modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ActivationStatus/LiferayExperienceCloud/components/SetupLXCModal/components/SetupLXCForm/components/SetupLXCPage/hooks/useSubmitLXCEnvironment.js

```diff
@@ -118,6 +119,43 @@ export default function useSubmitLXCEnvironment(
 						}
 					)
 				);
+
+				const emailProjectAdmin = lxcActivationFields?.admins
+					?.map(({email}) => email)
+					.join(', ');
+
+				const adminFirstNames = [];
+				const adminLastNames = [];
+
+				lxcActivationFields?.admins.forEach((admin) => {
+					const fullName = admin.fullName;
+					const formattedAdminFullName = fullName.split(' ');
+					const firstNameAdmin = formattedAdminFullName[0];
+					const lastNameAdmin = formattedAdminFullName
+						.slice(1)
+						.join(' ');
+
+					adminFirstNames.push(firstNameAdmin);
+					adminLastNames.push(lastNameAdmin);
+				});
+
+				const notificationTemplateService = new NotificationQueueService(
+					client
+				);
+
+				await notificationTemplateService.send(
+					'SETUP-LXC-ENVIRONMENT-NOTIFICATION-TEMPLATE',
+					{
+						'[%DATE_AND_TIME_SUBMITTED%]': new Date().toUTCString(),
+						'[%PROJECT_CODE%]': project.code,
+						'[%PROJECT_DATA_CENTER_REGION%]':
+							lxcActivationFields.primaryRegion,
+						'[%PROJECT_ID%]': lxcActivationFields.projectId,
+						'[%USER_EMAIL%]': emailProjectAdmin,
+						'[%USER_FIRST_NAME%]': adminFirstNames.join(', '),
+						'[%USER_LAST_NAME%]': adminLastNames.join(', '),
```

Comment:
@Aline160 from what I could understand from `lxcActivationFields?.admins` it's possible to have multiple `admins` right?

So it should send the email that way?

Suppose that I have as Admin: `Aline Pereira Liferay` and `Keven Leone dos Santos`

It should be:

**Project Admin(s)**

**Email:** keven.leone@mailnator.com, aline.pereira@mailnator.com
**First Name:**  Keven, Aline
**Last Name:** Leone dos Santos, Pereira Liferay

Or

Project Admin(s)

**Email:** keven.leone@mailnator.com
**First Name:**  Keven
**Last Name:** Leone dos Santos

**Email:**  aline.pereira@mailnator.com
**First Name:**  Aline
**Last Name:** Pereira Liferay

I used `Liferay` in our name because I don't know if you have a last name 😄

Reply:
Done. :)